src/Groups/group_constructors.jl

```diff
@@ -611,17 +611,17 @@ julia> gens(free_group([:a, :b], "x" => 1:2, 'y' => (1:2, 1:2)))
 ```
 """
 function free_group(L::Vector{<:Symbol}; eltype::Symbol = :letter)
-   @req allunique(L) "generator names must be unique"
+  @req allunique(L) "generator names must be unique"
    J = GapObj(L, recursive = true)
-   if eltype == :syllable
+  if eltype == :syllable
      G = FPGroup(GAP.Globals.FreeGroup(J; FreeGroupFamilyType = GapObj("syllable"))::GapObj)
```

Comment:
If the indentation gets changed from 3 to 2 spaces in a lot of places, why not also fo the lines that define `J` and `G`?